src/common/CMakeLists.txt

```diff
@@ -98,6 +108,8 @@ ecbuild_add_library( TARGET cloudsc-common-lib
         $<${HAVE_HDF5}:hdf5::hdf5_fortran>
         $<${HAVE_SERIALBOX}:Serialbox::Serialbox_Fortran>
         $<${HAVE_FIELD_API}:field_api_${prec}>
+    DEFINITIONS
+        ENABLE_CLOUDSC_MIX
```

Comment:
Is this intentionally always active, or should this actually be something like `$<${ENABLE_CLOUDSC_MIX}:ENABLE_CLOUDSC_MIX>`? (Just curious, because it would render the CPP guards in yoethf.F90, yomcst.F90 redundant)